sale_order_line_cancel/models/sale_order_line.py

```diff
@@ -55,6 +55,8 @@ def _check_moves_to_cancel(self, moves):
 
     def _update_qty_canceled(self):
         """Update SO line qty canceled only when all remaining moves are canceled"""
+        if self.env.context.get("ignore_sale_order_line_cancel", False):
```

Comment:
I would placed that here https://github.com/OCA/sale-workflow/blob/acdf3a9b5a2aa73f8dacf47ce557f6f92dbfef55/sale_order_line_cancel/models/stock_move.py#L15